File: elementary_computer_science/Python/game_number.py
t = int(input())
for _ in range(t):
	x, y, n = map(int, input().split())
	w = x*2**((n + 1) // 2)
	z = y*2**(n // 2)
	print(max(w, z) // min(w, z))

File: elementary_computer_science/Cpp/game_number.cpp
#include <iostream>
using namespace std;

int main() {
	int t, n, x, y;
	cin >> t;
	while (t--) {
		cin >> x >> y >> n;
		if (n % 2)
			x *= 2;
		cout << int(max(x, y) / min(x, y)) << '\n';
	}
}